cub/test/catch2_runner.cpp

```diff
@@ -0,0 +1,2 @@
+#define CUB_CONFIG_MAIN
```

Comment:
nit: would this need the copyright header?

Reply:
Nice catch!